samples/java/getting-started/webfrontend/.vscode/tasks.json

```diff
@@ -0,0 +1,22 @@
+{
+    "tasks": [
+        {
+            "label": "azds: prelaunch",
+            "command": "azds",
+            "args": [
+                "up",
+                "--port=57583:7000",
+                "--detach",
+                "--keep-alive",
+                "--",
+                "/javadebug/start.sh",
+                "mvn",
```

Comment:
Does this launch the Java app in the same manner as before? Waiting for a debugger to attach on port 8000?

Reply:
see reply in pom.xml